pegasus/sites.v3/code.org/public/private/k5survey/index.haml

```diff
@@ -22,119 +19,389 @@
-            count
-          end.reduce(:+) / students.count.to_f
-        else
-          lifetime = 0
-          levels = 0
-        end
+    individual_survey_totals = total_survey_data(individual_surveys)
+    all_surveys_totals = total_survey_data(all_surveys)
 
-        {
-          teacher_name: teacher[:name],
-          teacher_email: teacher[:email],
-          affiliate_name: affiliate[:name],
-          affiliate_email: affiliate[:user_id],
-          students_count: students.count,
-          students_average_lifetime_days: lifetime.round,
-          students_average_levels_completed: levels.round(2)
-        }
-      end.compact
-    end.compact.flatten
+    total_attendees = 0
+    workshops.each do |workshop|
+      data = JSON.parse(workshop[:data]).merge(JSON.parse(workshop[:processed_data]))
+      if data['total_attendee_count_i']
+        total_attendees += data['total_attendee_count_i']
+      end
+    end
   end
-%link(rel='stylesheet' type='text/css' href='/css/learn_tabs.css')
+
+:javascript
+  $(document).ready(function() {
+    $('select').bind("change", function () {
+      window.location = $("select option:selected").first().val();
+    });
+  });
+
 %h1
-  K5 Survey Statistics
-#learn-tabs{style: "padding-top:20px; padding-left:20px; padding-right:20px; background-color:white; background-color: rgb(219, 219, 219); overflow: hidden; margin-bottom: 10px;"}
-  %ul.nav.nav-tabs{style: "float: left;"}
-    %li.active.tab
-      %a{href:"/"} Results
-    %li
-      %a{href:"k5survey/raw2"} Raw2
+  K-5 Facilitator Profile
+%select
+  %option{value:"/private/k5survey"}
+    Select a Facilitator
+  - all_facilitators.each do |facilitator|
+    %option{value:"#{'/private/k5survey?user_id=' + facilitator[:user_id].to_s}"}
+      #{facilitator[:name]}
+
+- if workshops
+  %h2
+    #{workshops.first[:name]}
 
-%h2
-  Results
-%table
-  %tr
-    %th
-      email_s
-    %th
-      facilitator_prepared_i
-    %th
-      facilitator_knowledgeable_i
-    %th
-      facilitator_enjoyable_i
-    %th
-      facilitator_professional_i
-    %th
-      workshop_start_i
-    %th
-      workshop_end_i
-    %th
-      workshop_materials_i
-    %th
-      workshop_venue_i
-    %th
-      workshop_food_i
-    %th
-      satisfaction_s
 
-  - surveys.each do |row|
-    - data = JSON.parse(row[:data]).merge(JSON.parse(row[:processed_data]))
-    - workshop = fetch_workshop_info data['workshop_id_i']
-    - data['workshop_stopped_dt'] = workshop['stopped_dt']
-    - data['workshop_facilitator_email'] = workshop['user_id']
-    - data['workshop_facilitator_name'] = workshop['name_s']
-    - data['satisfaction_s'] = satisfaction_s_to_i(data['satisfaction_s'])
+- if params['user_id']
+  %table{style:'width:970px; layout:fixed;'}
+    %colgroup
+      %col{width: "400"}
+    %tr
+      %th{colspan:4}
+    %tr
+      %td
+        Facilitator name
+      %td
+        #{workshops.first[:name]}
```

Comment:
if this is for all workshops maybe it should be a list of all workshop names here?